modules/login/php/requestaccount.class.inc

```diff
@@ -223,6 +223,23 @@ class RequestAccount extends \NDB_Form
                 $err['captcha'] = 'Please complete the reCaptcha!';
             }
         }
+        // Send an email to the administrator informing them that a user has
+        // requested an account.
+        if (empty($err)) {
+            $config    = \NDB_Factory::singleton()->config();
+            $studyName = $config->getSetting('title');
+
+            \Email::send(
```

Comment:
Before doing this you should ensure that the config setting is a valid email. I think it's also risky to do this by default, including sandboxes and places that haven't set up their email.. especially since that's not the purpose of the 'From' config setting and the default is 'no-reply@example.com'.

I think we should add a new config setting which defaults to empty, and only send an email if it's explicitly enabled.

Reply:
Would this be two config settings then? e.g. `administratorEmail` and `emailAdminOnAccountRequest`?

Reply:
wouldn't something like `accountRequestContactEmail` be enough? If it's not set, there's no one to try emailing..

Reply:
In #5965 I used the `From` setting in another context to test whether the server can send emails. Also in #6076 I did the same thing.

It's probably useful to have a separate email where an administrator can be reached if `From` is not suitable. I think 2 is better.

Reply:
Blocked by #6101

Reply:
#6101 is merged